packages/components/src/components/primitives/Symbols/Symbols.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FC } from 'react';
+import { Kbd, KbdProps, useStyleConfig } from '@chakra-ui/react';
+
+export const Symbols: FC<KbdProps> = (props) => {
+  const styles = useStyleConfig('Symbols');
```

Comment:
Why do we need this?

Reply:
Because there is no `Symbols` component in Chakra, it's called Kbd.